packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -462,6 +475,87 @@ export default class OverTheBoxDetailsCtrl {
       });
   }
 
+  /**
+   * Get last seen values
+   */
+  getLastSeen() {
+    if (this.device && this.device.lastSeen) {
+      // check lastSeen access is less than 5 minutes
+      const currentDate = moment();
+      const lastSeen = moment(this.device.lastSeen);
+      const diffDate = currentDate.diff(lastSeen, 'minute');
+
+      this.lastSeenAccess = {
+        lastSeen: diffDate,
+        lastSeenHuman: moment.duration(currentDate.diff(lastSeen)).humanize(),
+      };
+
+      if (diffDate < this.OVERTHEBOX_DETAILS.lastSeen.limit) {
+        this.lastSeenAccess.isRecent = true;
+      } else {
+        this.lastSeenAccess.isRecent = false;
+      }
+    }
+  }
+
+  /**
+   * Check public IP to retrieve its status
+   */
+  checkPublicIP() {
+    return this.OvhApiIp.v6()
+      .query({
+        'routedTo.serviceName': this.serviceName,
+        type: 'overthebox',
+      })
+      .$promise.then((ips) => {
+        const [ip] = ips;
```

Comment:
If `ips` is not needed, I think we can 
```suggestion
      .$promise.then(([ip]) => {
```